src/map/mob.cpp

```diff
@@ -2239,7 +2239,7 @@ static TIMER_FUNC(mob_delay_item_drop){
  * Sets the item_drop into the item_drop_list.
  * Also performs logging and autoloot if enabled.
  * rate is the drop-rate of the item, required for autoloot.
- * flag : Killed only by homunculus?
+ * flag : Killed only by homunculus (1) or mercenary (2)?
```

Comment:
Makes no sense to make this a bitmask. It can only be killed by one type at the same time.
If this is the only information we use flag for, then maybe just hand over the BL type of the killer?